composer.json

```diff
@@ -43,6 +43,9 @@
         "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold"
     },
     "extra": {
+        "packagist.drupal-composer.org": {
```

Comment:
This bit of magic was just merged into drupal-packagist by @webflo. It will cause the packagist to completely ignore our make file, thus freeing us to make it look and work however we want.